common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_1905_vs.h

```diff
@@ -30,7 +30,7 @@ class tlvVsClientAssociationEvent : public BaseClass
 {
     public:
         tlvVsClientAssociationEvent(uint8_t* buff, size_t buff_len, bool parse = false);
-        tlvVsClientAssociationEvent(std::shared_ptr<BaseClass> base, bool parse = false);
+        explicit tlvVsClientAssociationEvent(std::shared_ptr<BaseClass> base, bool parse = false);
```

Comment:
The commit message is not very clear.

Reply:
Ok, I will explain the issue in more detail when I rebase.